modules/ROOT/pages/clauses/create.adoc

```diff
@@ -3,247 +3,142 @@
 [[query-create]]
 = CREATE
 
-[abstract]
---
-The `CREATE` clause is used to create nodes and relationships.
---
+== Introduction
 
+The `CREATE` clause allows you to create nodes and relationships.
+Similar to xref::clauses/match.adoc[`MATCH`], this is done using xref::patterns/index.adoc[patterns].
+However, the patterns you can use in `CREATE` are restricted to what is relevant to describe the newly created nodes and relationships.
 
 [[create-nodes]]
-== Create nodes
+== Syntax for nodes
 
-[[create-create-single-node]]
-=== Create single node
-
-Creating a single node is done by issuing the following query:
+The `CREATE` clause allows you to create one or more nodes.
+For each, you can specify labels and properties.
+You can give the nodes names that you can refer to later in the query.
+xref:clauses/where.adoc[`WHERE`-subclauses] and xref:clauses/match.adoc#label-expression-match-or-expression[label expression] are not allowed.
+Instead, multiple labels are separated by colons.
```

Comment:
This follows from my initial comment, `WHERE` only makes sense in a node pattern, not in a node.